### PR TITLE
 Add Item prompt

### LINKED ISSUE #8: 7. As a user, I want to see a welcoming prompt to add my first item if my list is empty to help me get oriented to how the app works.
AC:

* The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item

### PR DESCRIPTION
**Pull request by ciradu2204**
## Description
This PR displays a prompt in the ListView for users to add items to their list when there are no items on the list. The prompt is a message and a button which, when clicked, redirects the user to the AddView. 

## Related Issue
Closes #8 

## Acceptance Criteria

- [x] The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria

1. From your terminal pull down this branch with `git pull origin cz-ci-add-item-prompt` and check that branch out with `git checkout cz-ci-add-item-prompt`.
2. Then `npm start` to launch the app.
3. Create a new shopping list or join with an existing token.
4. If the list is empty, you should see the prompt displayed. On adding an item, the list should be displayed instead of the prompt. If all items added are deleted, the app should revert to displaying the prompt.

### CODE REVIEW

**Review by katia-utochkina — APPROVED** [review]

**Review by lamilekan263 — APPROVED** [review]

File: src/components/ListView.js [hunk 1]
```diff
@@ -1,11 +1,33 @@
...
+    if (loading === false && shoppingList[0] !== undefined) {
+      setLength(shoppingList[0].items.length);
+    }
+
+    if (length >= 1) {
+      setShoppingListEmpty(false);
+    } else {
+      setShoppingListEmpty(true);
+    }
+  }, [length, loading, shoppingList]);
```
Comment (alexlsalt) [inline src/components/ListView.js:21]: Really clear "empty state" logic here. Great readability!

File: src/components/ListView.js [hunk 2]
```diff
@@ -1,11 +1,33 @@
...
+
+  return loading === false && shoppingListEmpty ? (
+    <div className="prompt">
+      <h1>No Item Added</h1>
+      <Link to="/add-view" className="add-button">
+        Add Item
+      </Link>
+    </div>
```
Comment (alexlsalt) [inline src/components/ListView.js:29]: Looks great! That's definitely how I would do it 👍

**Review by alexlsalt — APPROVED** [review]
Great work here, team! Really great and readable code; and even though the issue itself wasn't too complicated, you really provided an elegant code solution. 

Looking forward to your demo this Sunday 💟

File: src/components/ListView.js (hunk 2 above)
Comment (C-Zekeri) [reply src/components/ListView.js:29]: Thank you Alex!

**Pull request merged**